RELEASE-NOTES.txt

```diff
@@ -22,6 +22,7 @@ Unreleased
 1.49.0
 ------
 * [*] Remove the cancel button from settings options (Android only) [https://github.com/WordPress/gutenberg/pull/29599]
+* [**] The media upload options of the Image, Video and Gallery block automatically opens when the respective block is inserted. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/2700]
```

Comment:
I'd move this line to `Unreleased` section since this changes haven't released yet.

Reply:
Good catch. Thanks! Seems merging in develop changed things up because it was under unreleased initially 7b8dcaf 😅